core/os/memory.h

```diff
@@ -62,6 +62,10 @@ class Memory {
 	static void *realloc_static(void *p_memory, size_t p_bytes, bool p_pad_align = false);
 	static void free_static(void *p_ptr, bool p_pad_align = false);
 
+	static void *alloc_aligned_static(size_t p_bytes, size_t p_alignment);
```

Comment:
Would be nice to have some comment with description of what is exactly allocated and how metadata is stored/aligned.

Reply:
Agreed

Reply:
Done.